src/coreComponents/physicsSolvers/fluidFlow/CompositionalMultiphaseFVM.cpp

```diff
@@ -469,9 +469,12 @@ real64 CompositionalMultiphaseFVM::scalingForSystemSolution( DomainPartition & d
 
   string const dofKey = dofManager.getKey( viewKeyStruct::elemDofFieldString() );
   real64 scalingFactor = 1.0;
-  real64 maxDeltaPres = 0.0, maxDeltaCompDens = 0.0, maxDeltaTemp = 0.0;
   real64 minPresScalingFactor = 1.0, minCompDensScalingFactor = 1.0, minTempScalingFactor = 1.0;
 
+  std::vector< std::tuple< real64, globalIndex > > maxRegionDeltaPresLoc;
```

Comment:
got a bit confused here - is there a real need to store all subregions values?

Reply:
probably not but there is a subregion loop .. so ??

Reply:
i was thinking something simple like that
```
if (subRegionData.localMaxDeltaPres < maxRegionDeltaPresLoc[1] )
  maxRegionDeltaPresLoc = {subRegionData.localMaxDeltaPres, localToGlobalMap[subRegionData.localMaxDeltaPresLoc};
}
```
?

Reply:
I'll simplify... but thought the max on tuple with lambda function  was pretty cool...